packit/copr.py

```diff
@@ -0,0 +1,172 @@
+import logging
```

Comment:
please rename this file to avoid possible import conflicts, since the official copr module is also named... copr